packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```
@@ -266,6 +268,25 @@ class ConfigScreenComponent extends React.Component<any, any> {
 					);
 				}
 
+				if (settings['sync.target'] === SyncTargetRegistry.nameToId('joplinServer')) {
+    				const goToJoplinServerLogin = async () => {
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)